client/components/Settings.vue

```diff
@@ -232,6 +234,39 @@ const videoQualities = ['144p', '240p', '360p', '720p', '1080p', '1440p', '2160p
           @valuechange="val => settingsStore.setDefaultVideoQuality(val.value)"
         />
       </div>
+      <div class="settings-number-menu">
+        <label for="video-speed-input">Default video speed</label>
+        <div :style="[settingsStore.videoSpeedAsList ? { paddingRight: '335px' } : { paddingRight: '315px' }]" style="display: flex;">
+          <label for="as-list" style="padding-right: 5px"> (as list ?)</label>
+          <CheckBox id="as-list"
+                    :value="settingsStore.videoSpeedAsList"
+                    :label="''"
+                    @valuechange="val => settingsStore.setVideoSpeedAsList(val)"
+          />
+        </div>
+        <input v-if="!settingsStore.videoSpeedAsList"
+               id="video-speed-input"
+               class="settings-number-input"
+               type="number"
+               name="video-speed"
+               :value="settingsStore.defaultVideoSpeed"
+               step="0.1"
+               max="3"
+               min="0.1"
+               @change="(e: any) => settingsStore.setDefaultVideoSpeed(parseInt(e.target.value))"
```

Comment:
```suggestion
               @change="(e: any) => settingsStore.setDefaultVideoSpeed(parseFloat(e.target.value))"
```

This is a fix for my mistake, just adding it here so it can be fixed with this PR.

Reply:
Okay, I'll add that too in the next commit